notification/Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM node:lts-alpine as builder
 
 WORKDIR /build
 COPY ./src ./src
+COPY ./tests ./tests
```

Comment:
unit tests in js typically are in src directory and follow the pattern `{fileName}.test.js`

Reply:
Are you sure about that? 
I did that because the folder tests was introduced beforehand in tsconfig.json.
![image](https://user-images.githubusercontent.com/56654666/85198180-cffe6780-b2e6-11ea-89f5-bca5b5fc7c7a.png)
I thought it looks more  structured because you can differentiate tests from the productive system more clearly (especially when you have more tests).

Another benefit would be that you can only copy the binaries in the src folder (dist/src) from the intermediate container

Reply:
So, these structural changes have to be discussed with the other developers of the ODS. If we want to make those changes, we should do it on every service then. However, this should *not* be part of this pull request and discussed separatly... the PR is already huge enough